### ISSUE TITLE
How to improve window dragging performance?

### ISSUE THREAD

**Issue by y2k04** [MicaForEveryone/MicaForEveryone#66]
I am experiencing quite a lot of lag when I drag windows around. Is there any way to fix this lag?

**Comment by Kenya-West** [comment]
Still no clue

**Comment by xxshady** [comment]
@y2k04 did you find the way to fix this?

**Comment by y2k04** [comment]
No I didn't. I just uninstalled it because of how painful the lag was to deal with on a daily basis. I just could not bare it anymore.

**Comment by y2k04** [comment]
I had closed this issue because nothing seemed to be done about it.

**Comment by dongle-the-gadget** [comment]
Did you turn on Blur Behind? If so, this is expected behaviour.

**Comment by y2k04** [comment]
No.

**Comment by xxshady** [comment]
> y2k04 did you find the way to fix this?

btw it was "blur behind" issue for me

**Comment by xmine64** [comment]
> I had closed this issue because nothing seemed to be done about it.

You're right, nothing can be done. If it happens only on a specific app then its the app's problem and you should make a rule to exclude the app. If this happens on all apps then it should be a bug in Windows or a hardware compatibility issue.

**Comment by y2k04** [comment]
> > I had closed this issue because nothing seemed to be done about it.
> 
> You're right, nothing can be done. If it happens only on a specific app then its the app's problem and you should make a rule to exclude the app. If this happens on all apps then it should be a bug in Windows or a hardware compatibility issue.

It occurs system-wide. All apps were lagging like crazy.

**Comment by y2k04** [comment]
It happened most notably with Explorer
